tests/client/disco/components/TestAddon.js

```diff
@@ -48,15 +48,9 @@ function renderAddon({ setCurrentStatus = sinon.stub(), ...props }) {
 
 describe('<Addon />', () => {
   describe('<Addon type="extension"/>', () => {
-    let root;
-
-    beforeEach(() => {
-      root = renderAddon(result);
-    });
-
     it('renders a default error overlay with no close link', () => {
       const data = { ...result, status: ERROR, setCurrentStatus: sinon.stub() };
-      root = renderAddon(data);
+      const root = renderAddon({ addon: data, ...data });
```

Comment:
All of these seem like evidence to suggest that `addon` doesn't need to be a property on `<Addon>` but that's an exercise for a future patch